flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/SequenceFileSerializerFactory.java

```diff
@@ -36,16 +38,14 @@
   @SuppressWarnings("unchecked")
   static SequenceFileSerializer getSerializer(String formatType,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "getSerializer" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)